packages/ERTP/src/types.js

```diff
@@ -18,21 +18,37 @@
  * portion.
  *
  * @property {Brand} brand
- * @property {Value} value
+ * @property {AmountValue} value
  */
 
 /**
- * @typedef {NatValue | SetValue} Value
- * Values describe a set or quantity of something that can be owned or shared.
- * Fungible values use a non-negative bigint to represent a quantity.
- * Non-fungible values use an array of `Key`s to represent a set of
- * whatever each key represents. A `Key` is a passable value that can
- * be used as an key an element in a set (SetStore or CopySet) or as the
+ * @typedef {NatValue | SetValue | CopySetValue} AmountValue
```

Comment:
This is a breaking change to the types of the ERTP package.  Please provide a deprecation, something like:

```js
/** @typedef {AmountValue} Value Deprecated alias for AmountValue. */
```

Reply:
Done